ci/infra/testrunner/tests/test_skuba_addon_upgrade.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import yaml
+import semantic_version
+import re
+from io import StringIO
+
+
+def decrease_version(version_str):
+    v = semantic_version.Version(version_str)
+    if v.patch > 0:
+        v.patch -= 1
+    elif v.minor > 0:
+        v.minor -= 1
+    elif v.major > 1:
+        v.major -= 1
+    else:
+        v.major = 0
+        v.minor = 1
+    return str(v)
```

Comment:
An idea, for testing purposes of these commands we could decrease to something like `0.0.1` so we don't need to really perform this decrease logic, after all we can perform the same checks regardless of what previous version we are mimicking, or is there an issue with this approach?

Reply:
You are right, this can be simplified, thanks!